lib/Widget/pre_operation/preTable.dart

```diff
@@ -10,295 +8,179 @@ class PreTable extends StatefulWidget {
+                              fontSize: 18,
+                              fontFamily: 'Prompt',
+                              color: Colors.black54,
+                              fontStyle: FontStyle.italic),
+                          columns: const <DataColumn>[
+                            DataColumn(
+                              label: Text(
+                                'HN',
+                              ),
+                            ),
+                            DataColumn(
+                              label: Text(
+                                'ชื่อ-นามสกุล',
+                              ),
+                            ),
+                            DataColumn(
+                              label: Text(
+                                'เพศ',
+                              ),
+                            ),
+                            DataColumn(
+                              label: Text(
+                                'อายุ',
+                              ),
+                            ),
+                            DataColumn(
+                              label: Text(
+                                'ห้อง',
+                              ),
+                            ),
+                            DataColumn(
+                              label: Text(
+                                'เตียง',
+                              ),
+                            ),
+                            DataColumn(
+                              label: Text(
+                                'ออกซิเจน',
+                              ),
+                            ),
+                            DataColumn(
+                              label: Text(
+                                'ความดัน',
+                              ),
+                            ),
+                            DataColumn(
+                              label: Text(
+                                'อุณหภูมิ',
+                              ),
+                            ),
+                            DataColumn(
+                              label: Text(
+                                'สถานะ',
+                              ),
+                            ),
+                          ],
+                          rows: const <DataRow>[
+                            DataRow(
+                              cells: <DataCell>[
+                                DataCell(Text('HN1234')),
+                                DataCell(Text('วริศรา จิ๋วประดิษฐ์กุล')),
+                                DataCell(Text('หญิง')),
+                                DataCell(Text('23')),
+                                DataCell(Text('1')),
+                                DataCell(Text('1')),
+                                DataCell(Text('98')),
+                                DataCell(Text('99')),
+                                DataCell(Text('43')),
+                                DataCell(Text('ปกติ')),
```

Comment:
How to inject the real data into this? Please help provide some point that might be ez to inject and let me prepare the needed data from firebase_service